src/sftp/handler_test.go

```diff
@@ -12,12 +11,108 @@ import (
 	"github.com/stretchr/testify/mock"
 	yekazip "github.com/yeka/zip"
 	"io"
-	"log/slog"
 	"os"
 	"path/filepath"
 	"testing"
 )
 
+func Test_NewSFTPHandler_UnableToGetSFTPServerPublicKeyNameSecret_ReturnsError(t *testing.T) {
+	buffer := utils.SetupLogger()
+
+	mockCredentialGetter := new(mocks.MockCredentialGetter)
+	mockCredentialGetter.On("GetSecret", mock.Anything).Return("test123", errors.New("error"))
```

Comment:
This mock scenario should probably return a `nil` or empty string for the first value, since the second value is an error (ditto for line 55 and anywhere else a mock setup returns an error)

Reply:
True, updated that!